calcom/cal.com
# fix: Self hosted url stuck at localhost:3000

## What does this PR do?

- Fixes [#8501](https://github.com/calcom/cal.com/issues/8501)

Adds a script that extracts the port from the NEXT_PUBLIC_WEBAPP_URL in the .env file and then passes it as a PORT env variable to the next cli.

Loom Video: https://www.loom.com/share/b52f75d3f51e4bacab23d0bfa0dc660a?sid=f2489b89-90c0-4a8d-88ec-f97923817d4f

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Update NEXT_PUBLIC_WEBAPP_URL and start the webapp to see it running on the new port
- If no port is specified it will default to 3000



- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
